MsixCore/msixmgr/msixmgrTelemetry.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <windows.h>
+#include <TraceLoggingProvider.h>
+
+TRACELOGGING_DECLARE_PROVIDER(g_MsixMgrTelemetryProvider);
+
+#define TraceLoggingOptionMicrosoftTelemetry() \
+    TraceLoggingOptionGroup(0x4f50731a, 0x89cf, 0x4782, 0xb3, 0xe0, 0xdc, 0xe8, 0xc9, 0x4, 0x76, 0xba)
+
+#define TelemetryPrivacyDataTag(tag) TraceLoggingUInt64((tag), "PartA_PrivTags")
+
+#define PDT_BrowsingHistory                    0x0000000000000002u
+#define PDT_DeviceConnectivityAndConfiguration 0x0000000000000800u
+#define PDT_InkingTypingAndSpeechUtterance     0x0000000000020000u
+#define PDT_ProductAndServicePerformance       0x0000000001000000u
+#define PDT_ProductAndServiceUsage             0x0000000002000000u
+#define PDT_SoftwareSetupAndInventory          0x0000000080000000u
+
+#define MICROSOFT_KEYWORD_CRITICAL_DATA 0x0000800000000000 // Bit 47
```

Comment:
Are all of these getting used anywhere? If not, then those not getting used should be removed.

Reply:
I don't think we should remove these tags even though most of them are not being used, because suppose in future someone wants to add additional telemetry and he/she might need these tags then he/she needs to unnecessarily bring all these tags again. These tags are in total all the tags that is needed for Telemetry.

Reply:
Done